src/components/lab/SpectrumBar.js

```diff
@@ -110,16 +143,24 @@ const SpectrumBar = ({ stageWidth, stageHeight }) => {
       {/* wavelength labels */}
       <Text
         text={t('Wavelength (nm)')}
-        // this one has to be positioned manually
-        x={spectrumBarInitialXPosition + TOTAL_SPECTRUM_BAR_WIDTH / 2 - 45}
+        fontSize={SPECTRUM_BAR_LABELS_FONT_SIZE}
+        ref={wavelengthLabel}
+        // this positions the label in the middle of the spectrum bar
+        x={
+          spectrumBarInitialXPosition +
+          TOTAL_SPECTRUM_BAR_WIDTH / 2 -
+          wavelengthLabelWidth / 2
+        }
+        // vertically, position it manually
         y={
           spectrumBarInitialYPosition +
           SPECTRUM_BAR_HEIGHT +
-          WAVELENGTH_LABELS_Y_AXIS_ADJUSTMENT_FACTOR
+          WAVELENGTH_LABELS_Y_AXIS_ADJUSTMENT_FACTOR * 5
```

Comment:
Why not include the `* 5` in the FACTOR?

Reply:
because other elements in the spectrum bar are using `WAVELENGTH_LABELS_Y_AXIS_ADJUSTMENT_FACTOR` without the `* 5` (these are the labels on the spectrum bar like 1000, 700, etc.). previously, all of these labels were just positioned with `y={... + WAVELENGTH_LABELS_Y_AXIS_ADJUSTMENT_FACTOR}`, but when we made the font bigger they could no longer fit on the same line.